app/src/main/kotlin/org/nixos/gradle2nix/Main.kt

```diff
@@ -80,14 +82,23 @@ class Main : CliktCommand(
         .projectDir()
         .default(File("."))
 
+    private val args: List<String> by option("--arg", "-a",
+            metavar = "ARGUMENT",
+            help = "Add an extra argument to the build.gradle")
+            .multiple()
+
+    private val ignoreMavenLocal: Boolean by option("--ignore-maven-local",
```

Comment:
Should this be reversed? I can't think of a good reason to include the `.m2` repository by default, as it hinders reproducibility.

So maybe this option can be named `includeMavenLocal`, and the argument value renamed as well. Then `PluginResolver` just reverses its logic to always ignore `MavenLocal` repositories unless the argument is true.